src/main/java/jenkins/plugins/mailer/tasks/MimeMessageBuilder.java

```diff
@@ -110,10 +118,27 @@ public MimeMessageBuilder setFrom(@Nonnull String from) {
     }
 
     public MimeMessageBuilder setReplyTo(@Nonnull String replyTo) {
-        this.replyTo = replyTo;
+        try {
```

Comment:
MimeMessageBuilder is single-thread, right? If no, the new code introduces concurrency issues